### DISCUSSION TITLE
Welcome to NavisAddinManager Discussions!

### DISCUSSION THREAD

**Discussion by chuongmep** [chuongmep/NavisAddinManager#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by miyuki136630** [comment]
What is the bottleneck in upgrading to 2025? What should I do? I tried it but it didn't work.

**Reply by chuongmep** [reply]
Hi @miyuki136630 , sorry for my late reply, yes you can upgrade to 2025 with configuration change, but need to test with the framework I think. I'm don't have so much time to help upgrade but will consider in my todo list !

**Reply by miyuki136630** [reply]
Thank you for completing it so quickly!
Navisworks 2025 is much better than 2024, so it's very helpful!
I was studying Navisworks add-ins and was very enthusiastic about them.
That's why. Sorry for the late reply.
It's really a big help to be able to debug it repeatedly. Thank you!
